pylint/checkers/similar.py

```diff
@@ -93,15 +376,32 @@ def run(self):
     def _compute_sims(self):
         """compute similarities in appended files"""
         no_duplicates = defaultdict(list)
-        for num, lineset1, idx1, lineset2, idx2 in self._iter_sims():
+        for (
+            num,
+            lineset1,
+            start_line_1,
+            end_line_1,
+            lineset2,
+            start_line_2,
+            end_line_2,
+        ) in self._iter_sims():
```

Comment:
This can easily cause issues if the return of `_iter_sims` changes slightly. It would be better use a `NamedTuple` for the return of `_find_common`.